browser/js/content.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+function init() {
+  // Only do work on reddit.
+  if (!location.hostname.endsWith('.reddit.com')) return;
```

Comment:
Non-www reddit will redirect to www, there are a bunch of other subdomains though like np.reddit.com and ssl.reddit.com. All the relevant ones should work.